lib/api_repo.js

```diff
@@ -806,57 +831,85 @@ function processPaths(done, process) {
  * Finds the paths to the proto files with the given api name and version.
  *
  * If callback is set, it calls back on each of them.
- * @param {string} name the api name.
```

Comment:
You can put a hyphen between the parameter name and the doc (http://usejsdoc.org/tags-param.html#names-types-and-descriptions)